tox.ini

```diff
@@ -9,7 +9,7 @@ depends =
     report: py{37,38,39,310}
 commands =
     python --version
-    pytest -vv --cov=taskgraph --cov=src/taskgraph/run-task --cov-append --cov-report= {posargs}
+    pytest -n auto -vv --cov=taskgraph --cov=src/taskgraph/run-task --cov-append --cov-report= {posargs}
```

Comment:
I'm not sold on the parallel test idea, but locally at least it saves multiple minutes per run, and doesn't seem to cause any issues. Thoughts?

Reply:
Fine by me! We can always revert this small change if we face too many intermittent errors.